packages/app-project/src/helpers/logger/logReactError.js

```diff
@@ -1,4 +1,4 @@
-import * as Sentry from '@sentry/browser'
+import * as Sentry from '@sentry/node'
```

Comment:
Why does `logReactError` load `@sentry/node`? I don't understand why this is defaulted to the node logger if this is only ever used client side by the classifier. It seems like this gets changed to use the browser version by the aliasing... so why not just load up the browser logger directly still?

Reply:
This is catching render errors during SSR.

Reply:
So, not logging a React error then? What is it doing in the classifier?

Reply:
Sorry, it catches render errors during SSR, but also catches render errors when components crash in the browser.